app/components/TodoList.js

```diff
@@ -70,45 +67,14 @@ export default class TodoList extends Component {
           <Text style={styles.rowText} numberOfLines={1}>{rowData.text}</Text>
         </View>
       </TouchableOpacity>
-    )
+    );
   }
   renderSeparator(sectionID: number, rowID: number) {
     return (
       <View key={`${sectionID}-${rowID}`} style={styles.separator} />
-    )
+    );
   }
   onPressRow(rowData: {}) {
-    this.props.toggleTodo(rowData.id)
+    this.props.toggleTodo(rowData.id);
   }
 }
-
-const styles = StyleSheet.create({
-  sectionHeader: {
-    backgroundColor: 'rgb(48,48,48)',
-    height: 48,
-    paddingHorizontal: 16,
-    flex: 1,
-    flexDirection: 'row',
-    alignItems: 'center'
-  },
-  sectionHeaderText: {
-    color: 'rgba(255,255,255,0.7)',
-    fontSize: 16
-  },
-  row: {
-    height: 48,
-    paddingHorizontal: 16,
```

Comment:
We need to be sure that we want to have styles outside component. There is a significant difference between RN and React. This isn't CSS as known from web. RN handles styles differently.